### DISCUSSION TITLE
Nous A1T Update Fails from minimal to DE

### DISCUSSION THREAD

**Discussion by Ziltoidium** [arendst/Tasmota#21481] [Q&A] [answered]
I recently updated my Nous A1T to the minimal Tasmota firmware 14.0.0.   After that i wanted to update to the german version, but it fails due to "Not enought space".  I used the images from [http://ota.tasmota.com/tasmota/release/](url)
Never had this issue before when starting from minimal images.

Any idea how to solve this?  Thanks!

![Bildschirmfoto vom 2024-05-24 09-06-15](https://github.com/arendst/Tasmota/assets/9767200/15de97c8-c785-4427-8bcf-2921c63ab374)

**Comment by sfromis** [comment]
When upgrading, you have to always use the OTA URL given on the page you link, not the one (without `.gz`) listed in the "Initial Firmware" column.

**Comment by Ziltoidium** [comment]
I took this  http://ota.tasmota.com/tasmota/release/tasmota-DE.bin.gz
Also the standard english one, but does not work.  Anything else i could check on command line?

**Reply by barbudor** [reply]
Please share a screen shot of the bottom info page where flash sections sizes are shown
Here are my nous

![Screenshot_2024-05-24-20-18-31-071_com android chrome-edit](https://github.com/arendst/Tasmota/assets/2996042/f418cb81-a298-4029-b646-bead26e0217d)

**Comment by Ziltoidium** [comment]
i fear i don't have this info page at minimal firmware.  Or how can i get this info site?

**Comment by Ziltoidium** [comment, marked as answer]
i did a factory reset and now it works. No idea why, but problem is solved.
